platform/podjs/rollup-plugin-gen-podjs/genPodjs.js

```diff
@@ -22,6 +22,10 @@ function copyPodJs(dest) {
     }
 
     fs.copyFileSync("node_modules/@polypoly-eu/podjs/dist/pod.js", `${dest}`);
+    fs.copyFileSync(
+        "node_modules/@polypoly-eu/podjs/dist/01f7ed1f3f7ccab75eb4.wasm",
```

Comment:
Unless I'm missing something, this file is missing?

Reply:
That could be it and explains why I cannot reproduce it locally. No idea how to get cypress to find the file. Could be named differently